error-503.html

```diff
@@ -7,125 +7,159 @@
page-section .grid-row .results__header--none{padding:0}}.page-section .grid-row .results__details{padding:0 32px 32px 32px;position:relative}.page-section .grid-row .results__details--nearby{padding-bottom:32px;padding-left:0;padding-right:0}.page-section .grid-row .results__details--nearby:after{background-color:#d9e1e2;content:"";display:block;height:4px;left:0;position:absolute;bottom:-2px;width:130px}.page-section .grid-row .results__details .callout{display:block}.page-section .grid-row .results__nearby{padding:16px}@media (min-width: 925px){.page-section .grid-row .results__nearby{padding:32px}}.page-section .grid-row .results__nearby-list{list-style:none;margin:32px 0 0 0;padding:0}.page-section .grid-row .results__name{margin-top:8px}.page-section .grid-row .results-block--none{background-color:#0072c6;color:#fff;margin-bottom:32px;padding:16px}@media (min-width: 925px){.page-section .grid-row .results-block--none{margin-bottom:48px}}.page-section .grid-row .results-block--none .results__header{padding:8px 8px 12px 8px}@media (min-width: 925px){.page-section .grid-row .results-block--none .results__header{padding:8px 16px 0 16px}}.page-section .grid-row .results-block--none .results__none-content{background:#fff;color:#000;padding:16px}@media (min-width: 925px){.page-section .grid-row .results-block--none .results__none-content{padding:32px}}.page-section .places>ol{padding:0;list-style:none}.page-section .places>ol,.page-section .places p{margin:20px 0}.link-back{border-bottom:1px solid #0b0c0c;color:#000000;display:inline-block;font-size:16px;font-weight:400;line-height:1.25;margin-top:16px;margin-bottom:32px;padding-left:14px;position:relative;text-decoration:none;text-transform:none}@media (min-width: 925px){.link-back{margin-top:0}}.link-back::before{border-bottom:5px solid transparent;border-right:6px solid #0b0c0c;border-top:5px solid transparent;content:"";display:block;height:0;left:0;margin-top:-6px;position:absolute;top:50%;width:0}.link-back:link,.link-back:visited,.link-back:hover,.link-back:active{color:#000000}.error{margin-right:15px;border-left:4px solid #b10e1e;padding-left:10px}.error>.form-control{border:4px solid #b10e1e}@media (min-width: 641px){.error{border-left:5px solid #b10e1e;padding-left:15px}}.error-summary{border:4px solid #338ed1;margin-bottom:45px;padding:15px 10px;background-color:#fff}@media (min-width: 641px){.error-summary{border:5px solid #338ed1;margin-top:0;margin-bottom:45px;padding:20px 15px 15px}}.error-summary:focus{outline:3px solid #ffbf47}.error-summary-heading{margin-top:0}.error-summary-list{color:#000;list-style-type:none;margin-top:0;padding:0}@media (min-width: 641px){.error-summary-list li{margin-bottom:5px}}.error-summary-list a{color:#b10e1e;font-weight:bold;text-decoration:underline}.error-summary p{margin-bottom:10px}.results-block{background-color:#0072c6;color:#fff;margin-bottom:32px;padding:16px}.u-nomargin{margin:0}.u-margintop5{margin-top:5px}.local-header--question{padding:0}
```

Comment:
There is a warning due to the use of `Unicode Private Use Area`
![screen shot 2017-12-04 at 15 54 16](https://user-images.githubusercontent.com/639706/33561769-79b565f2-d90b-11e7-87e3-5e18a609e56f.png)

Reply:
Slipped through the cracks. Removed.